setup.py

```diff
@@ -69,9 +40,7 @@
     name="xhydro",
     packages=find_packages(include=["xhydro", "xhydro.*"]),
     test_suite="tests",
-    tests_require=test_requirements,
```

Comment:
This field is deprecated and the `pytest` and `setuptools` maintainers suggest against using it. Good call.

More info: https://docs.pytest.org/en/latest/explanation/goodpractices.html#do-not-run-via-setuptools